### PR TITLE
Mkdir cmd

### PR DESCRIPTION
**Pull request by rossem**

### CODE REVIEW

File: rules/mkdir.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		CreateInterm bool   `short:"p"`
+		Verbose      bool   `short:"v"`
+	}
+	filteredTokens, err := flags.ParseArgs(&mkdirFlags, cmd.TokensWithoutBase)
+	if err != nil {
+		return "", err
+	}
+	var dirPaths = parse.GetFilePathsFromTokens(filteredTokens)
+
+	return rmBaseCommand + " -rf " + strings.Join(dirPaths, " "), nil
```
Comment (narensrini) [inline rules/mkdir.go]: Is it necessary to force remove a directory you just made?

Additionally, if we're just removing a newly made directory, we can just `rmdir`

File: rules/mv.go [hunk 2]
```diff
@@ -24,7 +22,7 @@ func buildMv(from string, to string) string {
 }
 
 func getUndoMv(cmd models.Command) (string, error) {
-	filteredTokens, err := flags.ParseArgs(&mvFlags, cmd.Tokens[1:])
+	filteredTokens, err := flags.ParseArgs(&mvFlags, cmd.TokensWithoutBase)
 	if err != nil {
 		return "", nil
```
Comment (narensrini) [inline rules/mv.go]: Can we return the error here?

Comment (rossem) [inline rules/mv.go]: woops, my bad

File: rules/mkdir.go (hunk 1 above)
Comment (rossem) [inline rules/mkdir.go]: Why is rmdir better?

We need force if they attach some permissions to their newly made directory, I mean I can parse that out (it's the `m` flag), but having -f always will work

**Comment by narensrini** [comment]
With all the new tests and stuff, could you rebase and move the tests over?

**Comment by rossem** [comment]
done

**Comment by narensrini** [comment]
good show :bear: :+1:

**Pull request merged**